.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file does not belong either. 

See this for how to ignore files https://www.atlassian.com/git/tutorials/saving-changes/gitignore

Pay close attention to the section "Ignoring a previously committed file". You'l need to follow it since these files are already committed.